playbooks/roles/deploy-osh/defaults/main.yml

```diff
@@ -2,7 +2,7 @@
 # defaults file for deploy-osh
 
 # Location of the kubeconfig file, fetched from velum UI.
-kubeconfig_file_path: "~/suse-osh-deploy/kubeconfig"
+kubeconfig_file_path: "{{ socok8s_workspace }}/kubeconfig"
```

Comment:
Can this use the common vars?

Reply:
Its already coming from a global shared variable defined in group-vars.

Reply:
I am fine with the reuse.